installer/dockerfile/webhook-manager/Dockerfile

```diff
@@ -15,8 +15,13 @@
 
 # The base image is created via `Dockerfile.base`, the base image is cached
 # since the required packages change very rarely.
+FROM golang:1.17 AS builder
+WORKDIR /go/src/volcano.sh/
+ADD . volcano
+RUN cd volcano && make vc-webhook-manager
+
 FROM volcanosh/vc-webhook-manager-base:1.3.1
```

Comment:
I guess here might be the reason of ci failed, `volcanosh/vc-webhook-manager-base:1.3.1` havn't supported multi-arch yet.

Reply:
Thanks yikun, I'll have a try.